eclair-core/src/main/scala/fr/acinq/eclair/blockchain/bitcoind/BitcoinCoreWallet.scala

```diff
@@ -34,16 +34,16 @@ class BitcoinCoreWallet(rpcClient: BitcoinJsonRPCClient)(implicit system: ActorS
   import BitcoinCoreWallet._
 
 
-  def fundTransaction(hex: String, lockUnspents: Boolean): Future[FundTransactionResponse] = {
-    rpcClient.invoke("fundrawtransaction", hex, BitcoinCoreWallet.Options(lockUnspents)).map(json => {
+  def fundTransaction(hex: String, lockUnspents: Boolean, feeRatePerKw: Long): Future[FundTransactionResponse] = {
+    rpcClient.invoke("fundrawtransaction", hex, Options(lockUnspents, satoshi2btc(Satoshi(feeRatePerKw)).amount)).map(json => {
```

Comment:
our fee rate is in satoshi/kiloweight, and bitcoind expects a fee rate in btc/kilobyte so you need to convert from kw to kb (there's a function for that in the eclair package)